authenticatorbridge/src/main/java/com/bitwarden/authenticatorbridge/util/EncryptionUtils.kt

```diff
@@ -180,7 +182,7 @@ private fun SharedAccountData.toJsonModel() = SharedAccountDataJson(
  */
 private fun SharedAccountDataJson.toDomainModel() = SharedAccountData(
     accounts = this.accounts.map { account ->
-        SharedAccountData.Account(
+        com.bitwarden.authenticatorbridge.model.SharedAccountData.Account(
```

Comment:
Is this package needed here?

Reply:
Good callout...it should not be, so hopefully this is just a quirk of the refactor tool.

Reply:
05e5806c019f3c93161698be88831c8cff9cf736 🤗